### PR TITLE
Update remote-resources for Jetty 11.

### PR DESCRIPTION
**Pull request by joakime**
+ Removed `LICENSE` as it refers to old EPL1
+ Replaced with `LICENSE.md` to conform with current Eclipse foundation norms, and include the content for EPL2 + ASL2
+ Renamed `NOTICE.txt` to `NOTICE.md` to conform to current Eclipse Foundation norms.
+ Updated content in `NOTICE.md`
+ Updated `about.html`, but I think it can be deleted as we are not "delivered as Eclipse Platform Plug-ins and Fragments" as outlined in https://www.eclipse.org/legal/epl/about.php

### CODE REVIEW

**Comment by joakime** [comment]
Note that this version is only really applicable for Jetty 11.
The contents of the NOTICE.md will be different in Jetty 12 (as it include all of ee8, ee9, and ee10)
I wouldn't mind pulling this into the Jetty 11 build itself, and not relying on jetty.toolchain for this content.

**Review by jmcc0nn3ll — CHANGES_REQUESTED** [review]
What happened to putting this into the relevant jetty projects and not using the remote artifact resource?

Also, I think we can get rid of the about.html. If it is an issue for the platform consuming jetty through their p2 repositories, they can tackle that problem.

**Comment by joakime** [comment]
> How about we just leave the `jetty-artifact-remote-resources` project alone, and ensure that we change any project that is using it to have it's own NOTICES/LICENCES file? I don't think it makes sense to have this kind of blanket approach to generating these files, considering how important they've become over the years.

This proposal is a way bigger hassle IMO.

That literally means we have `NOTICE.md` and `LICENSE.md` is copied to every single module's `src/main/resources/META-INF/` directory.
Keep in mind that these remote-resources are used by all of our produced (and deployed) artifacts, from things like jetty-util to things like apache-jsp.

File: jetty-artifact-remote-resources/src/main/resources/META-INF/NOTICE.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+  * The Apache 2.0 License (ASL2)
+    http://www.apache.org/licenses/LICENSE-2.0.html
+
+  * The Eclipse Public 2.0 License (EPL2)
+    https://www.eclipse.org/legal/epl-2.0/
+
+Jetty may be distributed under either license.
+
+```
+SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
```
[the remark this answers is not in the document]
Comment (joakime) [reply jetty-artifact-remote-resources/src/main/resources/META-INF/NOTICE.md:19]: Lets get the language right first.
Then we can merge it here and move it en-masse without changes in content to the locations where we need it (the jetty project produced artifacts in all of the various repos we have)

**Comment by janbartel** [comment]
> > How about we just leave the `jetty-artifact-remote-resources` project alone, and ensure that we change any project that is using it to have it's own NOTICES/LICENCES file? I don't think it makes sense to have this kind of blanket approach to generating these files, considering how important they've become over the years.
> 
> This proposal is a way bigger hassle IMO.
> 
> That literally means we have `NOTICE.md` and `LICENSE.md` is copied to every single module's `src/main/resources/META-INF/` directory. Keep in mind that these remote-resources are used by all of our produced (and deployed) artifacts, from things like jetty-util to things like apache-jsp.

Sheesh, that _is_ a PITA. So we need to maintain at least 4 different versions: for jetty-9, 10, 11, 12?

**Comment by janbartel** [comment]
I definitely think the `java-schemas` project needs to have it's own LICENCE and NOTICES files - all of the stuff about ow2, glassfish etc etc is just not applicable.

**Comment by joakime** [comment]
That's how I see it ...

Jetty 9
  - javax.[spec]
  - asm on old and new license (depends on asm version)
  - apache jsp 
  - apache jstl
  - old EE specs on:
    - CDDL + GPL2+CPE
    - BSD-3-Claus
    - ASL2  (eg: javax.security.auth.message 1.x)

Jetty 10
  - javax.[spec]
  - asm on new license
  - apache jsp / jstl
  - jakarta EE specs:
    - EPL2 + GPL2+CPE
    - BSD-3-Clause
    - ASL2 (eg: javax.security.auth.message 1.x)

Jetty 11
  - jakarta.[spec]
  - asm on new license
  - apache jsp
  - glassfish jstl
  - jakarta EE specs
    - EPL2 + GPL2+CPE (many specs in transition here, some moved to this, eg javax.security.auth.message 2.x)
    - BSD-3-Clause
    - ASL2 (but far less)

Jetty 12
  - asm on new license
  - _ee8_
    - javax.[spec] - Jakarta EE8 versions
      - CDDL + GPL2+CPE
      - BSD-3-Claus
      - ASL2  (eg: javax.security.auth.message 1.x)
    - apache jsp
    - apache jstl
  - _ee9_
    - jakarta.[spec]
      - CDDL + GPL2+CPE
      - BSD-3-Claus
      - ASL2  (eg: javax.security.auth.message 1.x)
    - apache jsp
    - apache jstl
  - _ee10_
    - jakarta.[spec]
      - CDDL + GPL2+CPE
      - BSD-3-Claus
      - ASL2  (but far fewer)
    - apache jsp
    - glassfish jstl
   
And that's just what I've discovered so far.

**Comment by joakime** [comment]
Closing, no longer relevant to this repo, or Jetty 11.

**Pull request closed without merging**